testrail/api.py

```diff
@@ -175,17 +175,21 @@ def _refresh(self, ts):
 
     @classmethod
     def flush_cache(cls):
-        """ Set all cache objects to refresh
+        """ Set all cache objects to refresh the next time they are accessed
         """
+        def clear_ts(cache):
+            if 'ts' in cache:
```

Comment:
I don't entirely remember why the check was there. I want to say there are instances where there can be a dict in the cache that doesn't have a `ts` key,  and this would avoid adding keys where they didn't previously exist.

I can remove the check if you like. Your call.